applications/values.yaml

```diff
@@ -10,150 +10,143 @@ lodestarStatusNamespaces: &statusNamespaces lodestar-frontend,lodestar-babylon-o
 # Version Manifest
 ##################
 applicationVersions:
-  frontend: &frontendVersion v1.0.4
-  backend: &backendVersion v1.0.5
-  git-api: &gitApiVersion v1.0.2
-  status: &statusVersion v1.0.0
-  resource-dispatcher: &resourceDispatcherVersion v1.0.9
-  agnosticv-operator: &agnosticvVersion helm_chart
-  anarchy-operator: &anarchyVersion helm_chart
-  poolboy: &poolboyVersion helm_chart
+  frontend: &frontendVersion v1.0.5
+  backend: &backendVersion v1.0.6
+  git-api: &gitApiVersion v1.0.3
+  status: &statusVersion master
```

Comment:
should this be `v1.0.0` instead of `master`?

Reply:
Updated